app/source/Kaidan.cpp

```diff
@@ -10,7 +10,7 @@
 Kaidan::Kaidan(NetworkFactories* networkFactories, QObject *parent) :
     rosterController_(new RosterController()), QObject(parent)
 {
-    client = new Swift::Client("jid@...", "pass", networkFactories);
+	client = new Swift::Client("jid@...", "pass", networkFactories);
```

Comment:
According to https://github.com/KaidanIM/Kaidan/issues/17 the majority voted for tabs